docs/source/listings.rst

```diff
@@ -182,9 +182,19 @@ Pub/Sub Alert Streams
       -
 
     * - lsst-alerts
-      - LSST alert stream in Pub/Sub, cleaned of duplicate alerts.
+      - Avro serialized LSST alert stream in Pub/Sub, cleaned of duplicate alerts.
         Messages contain the original alert bytes and metadata.
 
+    * - lsst-alerts-json
+      - JSON serialized LSST alert stream in Pub/Sub, cleaned of duplicate alerts.
+        Messages contain the original alert bytes and metadata.
```

Comment:
This one does not contain the original alert bytes. Besides the obvious (it's json not avro), I don't remember exactly what had to be changed for valid json, but add a sentence here describing that transformation. Doesn't have to list every change, but try to convey the important points.